azure-pipelines.yml

```diff
@@ -97,10 +97,12 @@ jobs:
     name: ''
     displayName: Install Signing Plugin
     enabled: True
+    condition: eq('${{ parameters.SignTypeToUse }}', 'real')
     inputs:
       signType: ${{ parameters.SignTypeToUse }}
       zipSources: false
   - task: MicroBuildLocalizationPlugin@3
+    condition: eq('${{ parameters.SignTypeToUse }}', 'real')
```

Comment:
Hi, just an idea we can keep the pipelines file single and reuse them for PR validations and PR with signed files. I think splitting them and creating duplicates can be avoided. You can refer VS extension pipelines for reference which does similar approach. Also, another point which Tats also pointed out is "test" sign the binaries for PR's instead of real signing. Is it worth to check with other teams what's the use of "test" sign vs "real" sign? so that we don't need to skip test signing.

we can also figure out the auth issue for installing signing plugin. My thought on the auth issue for installing signing plugin is to give access to GitHub service connection/account so that it is authorized to install the plugin at least for the test signs (we can give a try on this), for real signing agreed we need to go through the M2 approval process.

Reply:
Thanks Hari. I synced on this with David from DevDiv last week before sending this PR. It was his advice to separate pipelines (reply to one of tats comments on the why as well as PR description has more details). 

"test" is a mode to make sure everything is ready for signing but it wont use real valid certs and actually sign. The usage is to run as close to signing as possible but saving time since real signing is not done. I think this is for cases where signing takes a long time (normally big binaries to sign - singing step of the pipeline in one of my previous teams took 40+ min). This is not very useful to us since we have small binaries and signing step does not add that much time for us. So even if we move signing out to separate PR, I do not think we will use test option.

Reply:
resolving based on offline sync. Please feel free to re-open if needed.